### ISSUE TITLE
Add ability to add Alt Text when uploading an image

### ISSUE THREAD

**Issue by ShanaLMoore** [notch8/viva#418]
# Summary

When creating a new question through the form, if the user attaches an Image they should also be able to give it alt text. 

Currently the form does not give the user an option to attach the alt text

# Acceptance Criteria

- [ ] A user is able to set alt text on an Image when creating a new question
- [ ] the alt text should be visible in the html after being set
- [ ] a user cannot submit empty alt text if an image is uploaded

# Screenshot

![Image](https://github.com/user-attachments/assets/7d3aae91-b3e7-47c4-93a8-bedcc6b5834d)

![Image](https://github.com/user-attachments/assets/7c8f4973-5dab-47e2-9917-3158369fec1e)

# Testing Instructions

Create any question via the UI's form method
When uploading a question, alt text should be required
After submission, inspect the saved question's image. An alt tag should be present that's set to the submitted alt text. 

# Notes
